resources/js/components/IndexField.vue

```diff
@@ -10,7 +10,15 @@ export default {
     computed: {
         url() {
             return this.field.value
+        },
+        style() {
+            return {
+              backgroundImage: `url(${this.url})`,
```

Comment:
I think this might need to be written as `url('${this.url}')` to avoid any issues where the URL has funky characters in it